src/cli/java/org/commcare/util/screen/SessionUtils.java

```diff
@@ -20,9 +23,18 @@
 import java.net.PasswordAuthentication;
 import java.net.URL;
 
+import okhttp3.Credentials;
+import okhttp3.HttpUrl;
+import okhttp3.MultipartBody;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
 public class SessionUtils {
+    public SessionUtils() {}
 
-    public static void restoreUserToSandbox(UserSandbox sandbox,
+    public void restoreUserToSandbox(UserSandbox sandbox,
```

Comment:
It seems like we only changed it to non-static to handle tests. Curious if it's not possible to mock the static methods using Mockito ? That might make these changes redundant.

Reply:
It is possible to do with Mockito but it's not a great design pattern to rely on static functions.